oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -239,6 +239,9 @@ storage:
     group: ${SW_STORAGE_BANYANDB_GROUP:default}
     maxBulkSize: ${SW_STORAGE_BANYANDB_MAX_BULK_SIZE:5000}
     flushInterval: ${SW_STORAGE_BANYANDB_FLUSH_INTERVAL:15}
+    metricsShardsFactor: ${SW_STORAGE_BANYANDB_METRICS_SHARDS_FACTOR:2}
```

Comment:
All these new configurations are not added to https://github.com/apache/skywalking/blob/master/docs/en/setup/backend/configuration-vocabulary.md

Reply:
This doc should be updated as well

https://github.com/apache/skywalking/blob/master/docs/en/setup/backend/backend-storage.md